scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/package.scala

```diff
@@ -1,3 +1,5 @@
+package com.twitter.scalding
```

Comment:
Yes, but I wonder why the compile didn't fail before. Ahh, because it was a root package. :/ Good catch.